tests/vcs/git/GitCommitBearTest.py

```diff
@@ -125,10 +125,65 @@ def test_empty_message(self):
                          [])
         self.assert_no_msgs()
 
+    def test_auth_email_checks(self):
+        # Setting invalid user.email
+        # newgitdir = mkdtemp()
+        # os.chdir(newgitdir)
+        # self.run_git_command('init')
+        self.run_git_command('config', 'user.email invalid#!format!*!@email.com')
+
+        # Invalid email with no dns check
+        # emailChange = run_shell_command('git config user.email')
+        # self.assertEqual(emailChange[0][:-1], 'invalid#!format!*!@email.com')
+        self.git_commit('Shortlog\n\n'
+                        'some text\n'
+                        'Fix 2017')
+        self.assertEqual(self.run_uut(),
+                         ['Commit\'s author has invalid email.'])
+
+        # Setting back to original configuration
+        # os.chdir(self.gitdir)
+        # shutil.rmtree(newgitdir)
+        self.run_git_command('config', 'user.email coala@coala.io')
+
+        # Valid email with no dns check
+        self.git_commit('Shortlog\n\n'
+                        'some text\n'
+                        'Fix 2017')
+        self.assertEqual(self.run_uut(), [])
+
+        # Return Object for successful mock
+        class ReturnObject:
+            ERROR_CODES = {}
+
+
+        # Test successful DNS lookup
+        with unittest.mock.patch("bears.vcs.git.GitCommitBear.is_email", return_value=ReturnObject()):
```

Comment:
Line is longer than allowed. (102 > 79)

*LineLengthBear, severity NORMAL, section `linelength`.*